keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -714,12 +799,19 @@ message ProvisionerRevokeBucketAccessResponse {
 # Graduation Criteria
 
 ## Alpha
-
 - API is reviewed and accepted
 - Implement all COSI components to support Greenfield, Green/Brown Field, Brownfield and Static Driverless provisioning
 - Evaluate gaps, update KEP and conduct reviews for all design changes 
 - Develop unit test cases to demonstrate that the above mentioned use cases work correctly
 
+## Alpha -> Beta
+- Basic unit and e2e tests as outlined in the test plan.
+- Metrics in kubernetes/kubernetes for bucket create and delete, and granting and revoking bucket access.
+- Metrics in provisioner for bucket create and delete, and granting and revoking bucket access.
+
+## Beta -> GA
+- Stress tests to iron out possible race conditions in the controllers.
+- Users deployed in production and have gone through at least one K8s upgrade.
```

Comment:
PRR template has not been filled out. Please copy it from the KEP template: https://github.com/kubernetes/enhancements/blame/master/keps/NNNN-kep-template/README.md#L335

You will need to complete Upgrade/Downgrade Strategy, Version Skew Strategy, and the Alpha sections of the questionnaire.